doc/classes/ConfirmationDialog.xml

```diff
@@ -8,10 +8,10 @@
 		To get cancel action, you can use:
 		[codeblocks]
 		[gdscript]
-		get_cancel_button().pressed.connect(self.canceled)
+		get_cancel_button().pressed.connect(_on_canceled)
 		[/gdscript]
 		[csharp]
-		GetCancelButton().Pressed += Canceled;
+		GetCancelButton().Pressed += OnCanceled;
```

Comment:
CC @godotengine/dotnet to confirm this would be the appropriate name style for a signal callback.

Reply:
.NET conventions use `On*` to name methods used to raise events, not for callbacks. But our own documentation already does not follow this convention (e.g. https://docs.godotengine.org/en/latest/tutorials/scripting/c_sharp/c_sharp_signals.html). So I guess it is fine?